packages/sources/tradingeconomics/README.md

```diff
@@ -1,6 +1,6 @@
 # Chainlink External Adapter for Tradingeconomics
 
-![1.1.41](https://img.shields.io/github/package-json/v/smartcontractkit/external-adapters-js?filename=packages/sources/tradingeconomics/package.json)
+![2.0.4](https://img.shields.io/github/package-json/v/smartcontractkit/external-adapters-js?filename=packages/sources/tradingeconomics/package.json)
```

Comment:
Why doesn't this match the version from `package.json`?

Reply:
🤷 but I don't think it should hold up getting this hot-fix released.